crates/modelardb_client/src/main.rs

```diff
@@ -92,29 +94,40 @@ fn main() -> Result<(), String> {
     .map_err(|error| format!("Cannot execute queries: {error}"))
 }
 
-/// Parse the command line arguments in `args` and return a pair with the address of the server to
-/// connect to and the file containing the queries to execute on the server. If one of the command
-/// line arguments is not provided it is replaced with [`None`] in the returned pair.
-fn parse_command_line_arguments(mut args: Args) -> (Option<String>, Option<String>) {
+/// Parse the command line arguments in `args` and return a triple with the host of the server to
+/// connect to, the port to connect to, and the file containing the queries to execute on the
+/// server. If one of these command line arguments is not provided it is replaced with [`None`]."
+fn parse_command_line_arguments(mut args: Args) -> (Option<String>, Option<u16>, Option<String>) {
     // Drop the path of the executable.
     args.next();
 
     // Parse command line arguments.
     let mut host = None;
+    let mut port = None;
     let mut query_file = None;
 
     for arg in args {
         let metadata = fs::metadata(&arg);
         if metadata.is_ok() && metadata.unwrap().is_file() {
             // Assumes all files contains queries.
             query_file = Some(arg);
+        } else if arg.contains(':') {
+            // Assumes anything with : is host:port.
+            let host_and_port = arg.splitn(2, ':').collect::<Vec<&str>>();
+            host = Some(host_and_port[0].to_owned());
+            port = Some(
+                host_and_port[1]
+                    .parse()
+                    .map_err(|_| "for host:port port must be from 1 to 65535.")
```

Comment:
Maybe remove `for host:port` from this error message.  Maybe also change it to `port must be between 1 and 65535.` or something similar.

Reply:
Fixed.